tests/eclient/testdata/app_dns.txt

```diff
@@ -61,7 +61,7 @@ stdout 'Address:\s*172.26.11.1'
 # Repeat DNS resolution tests after purging (we had bugs affecting this scenario).
 message 'Purging app1'
 eden pod purge app1
-test eden.app.test -test.v -timewait 5m PURGING app1
+test eden.app.test -test.v -timewait 5m HALTING app1
 test eden.app.test -test.v -timewait 5m RUNNING app1
```

Comment:
@christoph-zededa looking at the logs and timestamps it looks like the check for RUNNING doesn't wait but immediately bails when it sees the HALTING state. See https://github.com/lf-edge/eve/actions/runs/9469623568/job/26093030065?pr=3975 which has:
```
        # check purging of app with large volume (607.540s)
        > eden pod purge eclient-mount
        [stdout]
        config changed, to see config run 'eden controller edge-node get-config'
        time="2024-06-11T18:43:57Z" level=info msg="app eclient-mount purge done"
        > test eden.app.test -test.v -timewait 2m HALTING eclient-mount
        testProg: /home/runner/actions-runner/_work/eve/eve/eden/dist/bin/eden.app.test
        [stdout]
        Docker app's state test
        === RUN   TestAppStatus
        apps: '[eclient-mount]' state: 'HALTING' secs: 120
        time: 2024-06-11T18:43:57.315265649Z out: 	appName eclient-mount state changed to RUNNING
        time: 2024-06-11T18:44:01.748449641Z out: 	appName eclient-mount state changed to HALTING
            testProc.go:84: time: 2024-06-11T18:44:01.748479667Z out: projects.ProcInfoFunc done with return: 
                time: 2024-06-11T18:44:01.748472864Z out: app eclient-mount state HALTING
                
            testContext.go:271: done for device c7bd04af-25f0-4161-8e08-8605e71bdee1
        --- PASS: TestAppStatus (5.44s)
        PASS
        [stderr]
        time="2024-06-11T18:44:01Z" level=info msg="time: 2024-06-11T18:44:01.748479667Z out: projects.ProcInfoFunc done with return: \ntime: 2024-06-11T18:44:01.748472864Z out: app eclient-mount state HALTING\n"
        > test eden.app.test -test.v -timewait 10m RUNNING eclient-mount
        testProg: /home/runner/actions-runner/_work/eve/eve/eden/dist/bin/eden.app.test
        [stdout]
        Docker app's state test
        === RUN   TestAppStatus
        apps: '[eclient-mount]' state: 'RUNNING' secs: 600
        time: 2024-06-11T18:44:02.776431251Z out: 	appName eclient-mount state changed to HALTING
            app_test.go:171: ASSERTION FAILED (2024-06-11T18:54:02.777237079Z): expected apps [eclient-mount] in RUNNING state
            app_test.go:173: 	actual eclient-mount: HALTING
            app_test.go:175: 	history of states for eclient-mount:
            app_test.go:177: 		state: no info from controller received in: 2024-06-11T18:44:02.76803218Z
            app_test.go:177: 		state: HALTING received in: 2024-06-11T18:44:02.776429067Z
```

Or 10 minutes isn't sufficient - perhaps we need to wait for longer?

Reply:
Today we found out that when running an edge application on EVE on the ubuntu github runner has an incredibly slow serial console (`time ls /usr/bin/` 3.94s versus 0.61s - serial console versus ssh).
So one thought we had was that the serial console slows down the startup of the whole edge application.

But that's just a theory that this is the reason for the eden tests to fail.

Reply:
Attached log indicates that app instance go to HALTING state (it takes 4 seconds), but do not return back from HALTING state (app hung in that state for 10 minutes until the test is over).